e2e/utils/infiniteScroll.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-console */
+const { scrollTo } = require('./scroll');
+
+const CARD_POSITIONER_TEST_ID = 'card-positioner';
+
+const triggerInfiniteScroll = async(browser)=>{	
+	const height = await browser.executeAsync(async(args, done) => {
+		const bodyHeight = document.body.scrollHeight;
+		
+		done(bodyHeight);
+	}, '');
+
+	await scrollTo(browser, height, 300);
+};
+
+const getCardPositionerStyles = async (browser, cardSelector)=>{
+	const cardPositionerStyle = await browser.executeAsync(async(id, cardPositionerTestId, done) => {
+		const getCardPositioner = (el) => {
+			const parentEl = el.parentElement;
+
+			if (parentEl.getAttribute('data-testid') !== cardPositionerTestId) {
+				return getCardPositioner(parentEl);
+			}
+
+			return parentEl;
```

Comment:
Just curiosity. In which case this will be different? When having multiple cards?

Reply:
yes, I prepared it for the multicard test